data-model/src/main/avro/RawSpan.avdl

```diff
@@ -16,4 +16,8 @@ protocol RawSpanProtocol {
 
     long received_time_millis = 0;
   }
+
+  record RawSpans {
```

Comment:
What's this for and why does it need to be defined as an avro object?

Reply:
It is used in RawSpansGrouper for aggregation. It is also serialized/deserialized to/from internal topics managed by kafka-streams. Refer to https://github.com/hypertrace/raw-spans-grouper/pull/17/files#diff-8d660452966db07a650ec3db094b5b19R91